### ISSUE TITLE
Should the virtual scrollbar/minimap follow the viewport when scrolling?

### ISSUE THREAD

**Issue by krassowski** [jupyterlab/jupyterlab#16518]
> For a long notebook, the virtual scrollbar does not fit on the page, and the scrollbar viewport disappears.
>
> <img src=https://github.com/jupyterlab/jupyterlab/assets/32258950/6123cd6a-521e-48dd-86e5-047f509adc3f width=400>
>
> Do you think the virtual scrollbar could be scrolled, to follow the scrollbar viewport ?
> 
> _Originally posted by @brichet in https://github.com/jupyterlab/jupyterlab/issues/16392#issuecomment-2182270125_

**Comment by krassowski** [comment]
>Do you think the virtual scrollbar could be scrolled, to follow the scrollbar viewport ?

Yes, I had the same thought and completely agree that this could be useful, but since this is an additional behaviour change (as of today the virtual scrollbar does not scroll when you move the viewport outside of it) I would defer it to a separate PR.

**Comment by JasonWeill** [comment]
I think it would be useful to scroll the minimap in this case, or, at the very least, to provide a small `v` or `^` indicator to tell the user that the active point on the scrollbar is out of view.
